packages/gsm8k/src/aviary/envs/gsm8k/env.py

```diff
@@ -108,8 +108,8 @@ async def step(
             False,
         )
 
-    def check_answer(self, answer: str) -> tuple[bool, float, Literal[True]]:
-        """Check if the proposed answer is correct.
+    def submit_answer(self, answer: str) -> tuple[bool, float, Literal[True]]:
+        """Submit the proposed answer and check if it is correct.
```

Comment:
Should the docstring mention that this tool is terminal? So that the agent knows